src/ProductlineApp.WebUI/ClientApp/src/pages/Settings.tsx

```diff
@@ -58,6 +60,22 @@ export default function Settings() {
     }
   };
 
+  const handleAvatarChange = async () => {
+    try {
+      const requestData = {
```

Comment:
napisalem w userService o co chodzi. Bledne cialo requesta to jest. Zeby to dzialalo musisz zrobic obiekt typu FormData i dodac do niego pole o nazwie 'avatar' z wartoscia, bedaca zaladowanym plikiem typu File